### ISSUE TITLE
React 18 support?

### ISSUE THREAD

**Issue by bonbonio** [jaketrent/react-social-icons#119]
Hello, just wondered if this is coming; the updated npm means that peer dependencies are stricter now.

**Comment by wadsack** [comment]
This would be great

**Comment by coeuscrypto** [comment]
Cool, will this be pushed to npm? 👍

**Comment by couetilc** [comment]
yes it will be pushed to NPM, I am dealing with some publishing issues and am getting in contact with the person who can resolve them

**Comment by couetilc** [comment]
This has been published in v5.14.0 https://www.npmjs.com/package/react-social-icons/v/5.14.0
